### DISCUSSION TITLE
Differences in Resolution of Videos

### DISCUSSION THREAD

**Discussion by jramborger78** [talmolab/sleap#1657] [General]
Hey guys. Potentially a silly question and obvious but figured I'd come to the source.

Our present model is quite good and doing everything we want it to. It was trained on 800x600 videos, but recently our parameters have changed to a resolution of 1280x720. I ran a tracking on one of these videos and it appeared to do well but not perfect, and was curious if, given they are the same environments between the resolutions, would it still perform adequately and independently, or would one have to:

A. Retrain from scratch with the new videos
B. Maybe add a video or 2 of the new resolution to fine tune it.

Thanks!

**Comment by talmo** [comment]
Hi @jramborger78,

Good question! By default, SLEAP will rescale it to the original resolution you trained it on. Technically, you can turn this off so that the scale is a bit more consistent, but we'd recommend going with your option B: training a model with a couple added videos with the new resolution.

Let us know how it goes!

Cheers,

Talmo
